From cd1a1abd976f29c77bf99ead3fc916e01e2c27d9 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Thu, 18 Dec 2014 11:59:07 -0500
Subject: [redhat] kabi: Adding symbol dev_warn (include/linux/device.h)

Message-id: <55fb2e1105308c6ad84d111eb136d658a4cfc9b2.1418903403.git.poros@redhat.com>
Patchwork-id: 102243
O-Subject: [PATCH RHEL7.1 15/48] kabi: Adding symbol dev_warn (include/linux/device.h)
Bugzilla: 1160751
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1160751
    https://bugzilla.redhat.com/show_bug.cgi?id=1160751

    Description:
    ============
    These are just conventient forms of printk(), if alternatives
    are considered preferable that isn't a big deal.
    Used in xpmem.ko

Symbol History
==============
 Request:
 ========
 [PATCH RHEL7 BZ1062669 37/49] kabi: Adding symbol dev_warn (include/linux/device.h)
 Jiri Olsa <jolsa@redhat.com> (Fri, 14 Feb 2014 14:23:59 +0100)
 Description:
 ============
 These are just conventient forms of printk(), if alternatives
 are considered preferable that isn't a big deal.

 Reviews:
 ========
 N/A

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/dev_warn
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64le/dev_warn
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/dev_warn
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/dev_warn

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/dev_warn b/redhat/kabi/kabi-module/kabi_ppc64/dev_warn
new file mode 100644
index 0000000..31ba5e1
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/dev_warn
@@ -0,0 +1,2 @@
+#1-
+0xfd2f64b3 dev_warn vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_ppc64le/dev_warn b/redhat/kabi/kabi-module/kabi_ppc64le/dev_warn
new file mode 100644
index 0000000..2f74a40
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64le/dev_warn
@@ -0,0 +1,2 @@
+#1-
+0x32da7972 dev_warn vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/dev_warn b/redhat/kabi/kabi-module/kabi_s390x/dev_warn
new file mode 100644
index 0000000..cf2120d
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/dev_warn
@@ -0,0 +1,2 @@
+#1-
+0x637c1723 dev_warn vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/dev_warn b/redhat/kabi/kabi-module/kabi_x86_64/dev_warn
new file mode 100644
index 0000000..3e29f4e
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/dev_warn
@@ -0,0 +1,2 @@
+#1-
+0x43b38448 dev_warn vmlinux EXPORT_SYMBOL
-- 
1.7.1